gr-digital/lib/costas_loop_cc_impl.cc

```diff
@@ -103,7 +99,7 @@ float costas_loop_cc_impl::phase_detector_8(gr_complex sample) const
        Circuits and Systems, Vol. 2, pp. 1447 - 1450, 2004.
     */
 
-    float K = (sqrt(2.0) - 1);
+    const float K = (sqrt(2.0) - 1);
```

Comment:
should be `sqrtf`, not `sqrt`. The back and forth between double and float has no upside, and sadly can't be optimized away.

Reply:
could you fix that in a separate commit, please?